sklearnex/neighbors/knn_unsupervised.py

```diff
@@ -22,11 +22,12 @@
 from onedal.neighbors import NearestNeighbors as onedal_NearestNeighbors
 
 from .._device_offload import dispatch, wrap_output_data
+from ..utils import get_namespace
 from .common import KNeighborsDispatchingBase
 
 
 @control_n_jobs(decorated_methods=["fit", "kneighbors"])
-class NearestNeighbors(sklearn_NearestNeighbors, KNeighborsDispatchingBase):
+class NearestNeighbors(KNeighborsDispatchingBase, sklearn_NearestNeighbors):

```

Comment:
does order make a difference?

Reply:
Yeah, the inheritance of kneighbors_graph will otherwise default to sklearn's version due to the MRO.